android/KMAPro/kMAPro/build.gradle

```diff
@@ -30,18 +33,9 @@ android {
             }
         }
 
-        String env_build_number = System.getenv("BUILD_NUMBER") //assigned by TeamCity, so all caps
-        String env_build_counter = System.getenv("build_counter")
-        if (env_build_number != null) {
-            println "Using build number $env_build_number and counter $env_build_counter from environment"
-            versionCode env_build_counter as Integer
-            versionName env_build_number
-        } else {
-            String majorMinorVersion = file('$projectDir/../../../../resources/VERSION.md').text.trim()
-            versionCode 100
-            versionName majorMinorVersion + '.0.0'
-            println "Using build $majorMinorVersion from project VERSION.md"
-        }
+        // VERSION_CODE and VERSION_NAME from version.gradle
+        versionCode VERSION_CODE as Integer
+        versionName VERSION_NAME
```

Comment:
Should line 86 be updated to get the tier from `TIER.md`?
```
switch (System.env.TIER) {
```

Reply:
Will check

Reply:
The `TIER` env var is still set as part of the build so this will continue to work. We can plan to do more refactoring as we go.